app/app/settings.py

```diff
@@ -74,12 +74,21 @@
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.RemoteUserMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'social_django.middleware.SocialAuthExceptionMiddleware',
     'applicationinsights.django.ApplicationInsightsMiddleware',
+    'custom_auth.CustomSSOMiddleware', 
 ]
 
+# added for custom SSO 
```

Comment:
The setting names should be renamed to be descriptive enough not to need the comment.